src/themes/material/elements/button.variables

```diff
@@ -25,7 +25,7 @@
 
 @borderBoxShadowColor: @solidBorderColor;
 @borderBoxShadow: 0 0 0 1px @solidBorderColor inset;
-@shadowBoxShadow: 0 0 0 0 transparent;
+@shadowBoxShadow: 0 transparent;
```

Comment:
according to https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow this is invalid. (and using chrome dev tools confirms that)
At least two length values have to be given, so you should change this to
```suggestion
@shadowBoxShadow: 0 0 transparent;
```

Reply:
fixed, I did not intended to change box shadow values